frontend/test/__support__/ui.tsx

```diff
@@ -154,12 +155,12 @@ function MaybeDNDProvider({
   );
 }
 
-export function getIcon(name: string) {
-  return screen.getByLabelText(`${name} icon`);
+export function getIcon(name: ByRoleMatcher, role: ByRoleMatcher = "img") {
+  return screen.getByRole(role, { name: `${name} icon` });
```

Comment:
@kulyk Worth pointing out that I swap the implementation to `*ByRole`, and all tests still pass (at least locally need to wait and see the CI build too). So I guess that's better since we can target icon buttons now.